easybuild/easyblocks/generic/bundle.py

```diff
@@ -65,7 +65,7 @@ def extra_options(extra_vars=None):
         })
         return EasyBlock.extra_options(extra_vars)
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, *args, check_for_sources=True, **kwargs):
```

Comment:
Since we are passing along kwargs all the time, i figured i could use it to put in an option to opt out of the source and patch check instead. But i fear this is python3 only syntax.

Reply:
It didn't work with python2, i was forced to make is a positional argument instead, but since we never really use these args for things anyway (except maybe in bundles? and bundles can't be bundled) that should be just fine.